examples/src/main/python/pi.py

```diff
@@ -27,12 +27,16 @@
 if __name__ == "__main__":
     """
         Usage: pi [partitions]
+
+        Monte Carlo method is used to estimate Pi in the below example.
     """
     spark = SparkSession\
         .builder\
         .appName("PythonPi")\
         .getOrCreate()
 
+    # If no arguments are passed(i.e. `len(sys.argv) < = 1` )
```

Comment:
Missing space after passed, problem in "< ="
This comment just restates the code below.